src/lib/abacus/logger.ts

```diff
@@ -19,6 +19,15 @@ class AbacusLogger implements Omit<AbacusLoggingProtocol, '__doNotUseOrImplement
       console.error(`${tag}: ${message}`);
     }
   }
+
+  ddInfo(tag: string, message: string, context?: {}) {
```

Comment:
👍 looks good to me, i have [another dd PR](https://github.com/dydxprotocol/v4-web/pull/932) which can come in after this but for now this is fine!